### PR TITLE
EPMRPP-96345 || EPMRPP-96148 || EPMRPP-96179 || EPMRPP-96185 || EPMRPP-96146 || EPMRPP-96147 || Fix the problems with the organization card

### PR DESCRIPTION
**Pull request by BlazarQSO**
## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const OrganizationCard = ({ organization }) => {
   const projectsCount = organization.relationships.projects.meta.count;
   const lastLaunch = organization.relationships.launches.meta.last_occurred_at;
```
Comment (AmsterGet) [inline app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx:56]: May be rename to `lastLaunchDate`?

File: app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ import { messages } from '../../messages';
 import styles from './organizationCard.scss';
 
 const cx = classNames.bind(styles);
-const THREE_MONTHS = 3600 * 24 * 30 * 1000;
+const THREE_MONTHS = 1000 * 60 * 60 * 24 * 30 * 3;
```
Comment (AmsterGet) [inline app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx:44]: May be rename to `THREE_MONTHS_IN_MS`?

**Review by AmsterGet — DISMISSED** [review]

File: app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx (hunk 2 above)
Comment (BlazarQSO) [reply app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx]: Done.

File: app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx (hunk 1 above)
Comment (BlazarQSO) [reply app/src/pages/instance/organizationsPage/organizationsPanelView/organizationCard/organizationCard.jsx]: Done.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**